doc/classes/Vector3.xml

```diff
@@ -123,7 +123,10 @@
 			<argument index="0" name="b" type="Vector3">
 			</argument>
 			<description>
-				Returns the dot product with [code]b[/code].
+				Returns the dot product with vector [code]b[/code]. This can be used to compare the angle between two vectors. For example, this can be used to determine whether an enemy is facing the player.
+				The dot product will be [code]0[/code] for a straight angle (90 degrees), greater than 0 for angles narrower than 90 degrees and lower than 0 for angles wider than 90 degrees.
+				When using unit vectors, the result will always be between [code]-1[/code] (180 degree angle) and [code]1[/code] (0 degree angle).
+				[b]Note:[/b] [code]a.dot(b)[/code] is equivalent to [code]b.dot(a)[/code].
```

Comment:
Does this new description apply to Vector3 like it does to Vector2? I haven't thoroughly checked this.

Reply:
Yes.